server/src/controllers/trip/editTrip.ts

```diff
@@ -34,7 +34,7 @@ export function editTripRoute() {
   return publicProcedure.input(validator.editTrip).mutation(async (opts) => {
     const { id, ...rest } = opts.input;
     const tripClass = new Trip();
-    const trip = await tripClass.update(rest,id);
-    return trip 
+    const trip = await tripClass.update(rest, id);
```

Comment:
Not sure why you are touching trip related services in this PR

Reply:
Trip and template related services were left so I tried changing them. I asked from Andrew about it.